docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -822,6 +822,46 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  '/event-types/{name}/partitions-number':
+    put:
+      tags:
+        - repartitioning-api
+      description: |
+        Repartitioning given event type to the number of provided paritions.
+
+      parameters:
+        - name: name
+          in: path
+          description: EventType name
+          type: string
+        - name: X-Flow-Id
+          in: header
+          description: |
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+        - name: partitions_number
+          in: body
+          schema:
+            $ref: '#/definitions/PartitionsNumber'
```

Comment:
`#/definitions/PartitionsNumber` is missing in definitions section.

Reply:
thank you, fixed here 91ea632b2 👍